specification/compute/resource-manager/Microsoft.Compute/stable/2019-12-01/compute.json

```diff
@@ -7535,11 +7541,27 @@
         "modelAsString": true
       }
     },
+    "DiffDiskPlacement": {
+      "type": "string",
+      "description": "Specifies the ephemeral disk placement for operating system disk.",
```

Comment:
Pls. add explanation what the different DiffDiskPlacement options mean.

Reply:
The x-ms-enum extension allows specifying a description for each enum value. (It helps generated docs.).

---
In reply to: [399608078](https://github.com/Azure/azure-rest-api-specs/pull/8847#discussion_r399608078) [](ancestors = 399608078)